pubspec.yaml

```diff
@@ -12,28 +12,28 @@ dependencies:
     sdk: flutter
 
   # From fluttercommunity.dev: Get current device information from within the Flutter application.
-  device_info_plus: ">=3.2.0 <4.2.0"
+  device_info_plus: ^6.0.0
```

Comment:
@Zazo032 Are you suggesting that we drop support for 3.2.0?

Reply:
What about supporting ">=4.0.0 <7.0.0"?

Reply:
That could work too. Just suggested 6.0.0 because there should be no behaviour/breaking changes to this plugin between v4 and v6.